src/uct/ib/dc/dc_mlx5_ep.h

```diff
@@ -447,8 +447,8 @@ static inline ucs_status_t uct_dc_mlx5_iface_dci_get(uct_dc_mlx5_iface_t *iface,
         if (uct_dc_mlx5_iface_dci_has_tx_resources(iface, ep->dci)) {
             return UCS_OK;
         } else {
-            txqp = &iface->tx.dcis[ep->dci].txqp;
-            UCS_STATS_UPDATE_COUNTER(txqp->stats, UCT_RC_TXQP_STAT_QP_FULL, 1);
+            UCS_STATS_UPDATE_COUNTER(iface->tx.dcis[ep->dci].txqp.stats,
```

Comment:
do we still need txqp variable?
compiler would warn it's unused..

Reply:
@yosefe 

`txqp` is used below in the code

Reply:
btw, we can remove easily it
just directly use `iface->tx.dcis[ep->dci].txqp`

`available = uct_rc_txqp_available(txqp);` -> `available = uct_rc_txqp_available(&iface->tx.dcis[ep->dci].txqp);`

Reply:
@yosefe wdyt?